README.rst

```diff
@@ -1,3 +1,30 @@
+Magic Wormhole progress
+=======================
+Magic Wormhole implementation now works with both Python 2 and Python 3.
+
+It is required the latest git version of Magic Wormhole.
```

Comment:
This will, of course, not always be true.  I think we need to wait with merging until there is a release on pypi. Do you know what version we will need to wait for?
Also: I think we don't need this modification to the README file. Rather, we want to indicate the new dependency in the installation instructions.